pact/policy-manager/policy-manager.pact

```diff
@@ -185,79 +145,95 @@
       receiver:string
       amount:decimal )
     (enforce-ledger)
-    (let ((policies:object{token-policies}  (at 'policies token)))
-      (map-transfer token sender guard receiver amount
-        (merge-policies-list policies))))
+    (let ((policies:[module{kip.token-policy-v2}]  (at 'policies token)))
+      (map-transfer token sender guard receiver amount policies)))
 
-  (defun enforce-crosschain:[bool]
-    ( token:object{token-info}
-      sender:string
-      guard:guard
-      receiver:string
-      target-chain:string
-      amount:decimal )
-    (enforce-ledger)
-    (let ((policies:object{token-policies}  (at 'policies token)))
-      (map-crosschain token sender guard receiver target-chain amount
-        (merge-policies-list policies))))
 
-  (defun enforce-withdraw:bool
-    ( token:object{token-info}
-      seller:string
-      amount:decimal
-      sale-id:string )
-    ;;TODO
-    true
+;; Sale/Escrow Functions
+  (defun get-escrow-account (sale-id:string)
+    { 'account: (create-principal (create-capability-guard (QUOTE_ESCROW sale-id)))
+    , 'guard: (create-capability-guard (QUOTE_ESCROW sale-id))
+    })
+
+  (defun enforce-sale-pact:bool (sale:string)
+    "Enforces that SALE is id for currently executing pact"
+    (enforce (= sale (pact-id)) "Invalid pact/sale id")
+  )
+
+
+  (defun create-concrete-policy:object{concrete-policy} (policies:[module{kip.token-policy-v2}])
```

Comment:
In line with the previous comment, I feel we could move this to the util-v1 file